src/components/SendActivityModal/SendActivityModal.tsx

```
@@ -174,7 +174,11 @@ const SendActivityModal = ({
    */
   const handleModeSelect = useCallback(
     (mode: ActivityMode) => {
-      store.setFormData({ mode });
+      const update: Partial<SendActivityFormData> =
+        mode === ActivityMode.PRESENCIAL
+          ? { mode, canRetry: false }
+          : { mode };
```

ESLint

[error] Replace `⏎··········?·{·mode,·canRetry:·false·}⏎·········` with `·?·{·mode,·canRetry:·false·}`

(prettier/prettier)